### PR TITLE
Added possibility to set selected item

### PR DESCRIPTION
**Pull request by mariomurrent-softwaresolutions**

### CODE REVIEW

File: lib/RNSingleSelect.tsx [hunk 1]
```diff
@@ -62,6 +62,7 @@ interface IProps {
   data?: Array<ISingleSelectDataType>;
   onTextChange?: (text: string) => void;
   onSelect: (selectedItem: ISingleSelectDataType) => void;
+  value?: ISingleSelectDataType;
```
Comment (kuraydev) [inline lib/RNSingleSelect.tsx]: I think we need to rename this to `initial`

File: lib/RNSingleSelect.tsx [hunk 2]
```diff
@@ -90,13 +91,18 @@ const RNSingleSelect = (props: IProps) => {
...
     spinnerSize = 30,
     spinnerColor,
+    value = null
     searchEnabled = true
   } = props;
 
+  const getInitialSelectedItemValue = () => {
+    return value ? value : null;
+  };
+
```
Comment (kuraydev) [inline lib/RNSingleSelect.tsx]: Actually, we do not need this function, its default value is already set to `null` which's why this is redundant.

**Review by kuraydev — CHANGES_REQUESTED** [review]

File: lib/RNSingleSelect.tsx (hunk 2 above)
Comment (mariomurrent-softwaresolutions) [reply lib/RNSingleSelect.tsx]: Yeah, you're totally right

**Review by kuraydev — APPROVED** [review]

**Comment by kuraydev** [comment]
@mariomurrent-softwaresolutions I will release a new version :) Thank you so much sir!

**Comment by mariomurrent-softwaresolutions** [comment]
You're welcome. Thx for letting me participate

**Comment by kuraydev** [comment]
@mariomurrent-softwaresolutions Please try the 0.3.3 version of the library :) I made it a bit better for the UX

**Comment by mariomurrent-softwaresolutions** [comment]
@WrathChaos Nice. thx

**Pull request merged**